x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -45,119 +51,102 @@ export interface IOverviewValues extends IOverviewServerData {
   flashMessages: IFlashMessagesProps;
 }
 
-interface IListenerParams {
-  actions: IOverviewActions;
-}
-
-interface IOverviewLogic extends IKeaLogic, IListenerParams {
-  values: IOverviewValues;
-}
-
 export const OverviewLogic = kea({
   actions: (): IOverviewActions => ({
-    setServerData: (serverData: IOverviewServerData) => serverData,
-    setFlashMessages: (flashMessages: IFlashMessagesProps) => ({
-      flashMessages,
-    }),
-    setHasErrorConnecting: (hasErrorConnecting: boolean) => ({ hasErrorConnecting }),
-    initializeOverview: ({ http }: { http: HttpSetup }) => ({ http }),
+    setServerData: (serverData) => serverData,
+    setFlashMessages: (flashMessages) => ({ flashMessages }),
+    setHasErrorConnecting: (hasErrorConnecting) => ({ hasErrorConnecting }),
+    initializeOverview: ({ http }) => ({ http }),
   }),
-  reducers: () => ({
+  reducers: (): TKeaReducers<IOverviewValues, IOverviewActions> => ({
     organization: [
-      {},
+      {} as IOrganization,
       {
-        setServerData: (_: IOverviewValues, { organization }: IOverviewValues) => organization,
+        setServerData: (_, { organization }) => organization,
```

Comment:
Same here, if you try doing something incorrect like
```suggestion
        setServerData: (_, { organization }) => false,
```

You'll see `Type 'boolean' is not assignable to type 'IOrganization'.`